packages/core-database-sequelize/lib/repositories/transactions.js

```diff
@@ -75,9 +73,7 @@ module.exports = class TransactionsRepository extends Repository {
     if (params.senderId) {
       const senderPublicKey = this.__publicKeyfromSenderId(params.senderId)
 
-      if (senderPublicKey) {
-        params.senderPublicKey = senderPublicKey
-      }
+      params.senderPublicKey = senderPublicKey
```

Comment:
Since the condition has been removed, then you could assign the value directly to `params.senderPublicKey`